src/app/showcase/pages/toolbar/toolbardemo.ts

```diff
@@ -19,6 +20,11 @@ export class ToolbarDemo {
             label: 'Basic',
             component: BasicDoc
         },
+        {
+            id: 'template',
+            label: 'Template',
```

Comment:
Maybe you can add a description for template section like the others sections.

![image](https://github.com/primefaces/primeng/assets/19764334/54fa1aa6-2204-4eb8-89f1-a9d4ff440f6e)

Reply:
I have added a description, I hope it is ok now. Thanks for reviewing!